tests/integration/local/start_api/test_start_api.py

```diff
@@ -369,6 +369,18 @@ def test_binary_request(self):
         self.assertEquals(response.headers.get("Content-Type"), "image/gif")
         self.assertEquals(response.content, input_data)
 
+    def test_request_with_form_data(self):
+        """
+        Form-encoded data should be put into the Event to Lambda
+        """
+        response = requests.post(self.url + "/echobase64eventbody",
```

Comment:
I know this endpoint echoes the Event back but the path is specific to base64. For this case it wont affect the outcome but to keep things clean and readable, can you create a new path on the [EchoEventFunction](https://github.com/awslabs/aws-sam-cli/blob/develop/tests/integration/testdata/start_api/template.yaml#L37) and update the path here.